data-prepper-plugins/otel-proto-common/src/main/java/org/opensearch/dataprepper/plugins/otel/codec/OTelLogsFormatOption.java

```diff
@@ -14,7 +14,8 @@
 import java.util.stream.Collectors;
 
 public enum OTelLogsFormatOption {
-    JSON("json");
+    JSON("json"),
+    PROTOBUF("protobuf");
```

Comment:
Is `protobuf` the right term here? OTLP supports JSON for protobuf. Should this be `binary`? Or `binary_protobuf`?

https://opentelemetry.io/docs/specs/otlp/#json-protobuf-encoding

Reply:
I do not think `binary` is needed. In OTEL world, `otlp_json` and `otlp_protobuf` are standard terminology.